src/controllers/expense.controller.js

```diff
@@ -0,0 +1,132 @@
+const expensesHelpers = require('../helpers/expense.helpers');
+const expensesService = require('../services/expense.service');
+const {
+  INTERNAL_SERVER_ERROR,
+  EXPENSE_NOT_FOUND_ERROR,
+} = require('../utils/config');
+
+const create = async (req, res) => {
+  try {
+    const {
+      userId,
+      spentAt,
+      title,
+      amount,
+      note,
+      category = 'Other',
+    } = req.body;
+
+    const requestData = {
+      userId,
+      title,
+      amount,
+      category,
+      note,
+      res,
+    };
+
+    if (
+      expensesHelpers.isUserExist(userId, res) ||
+      expensesHelpers.validateRequestData(requestData)
+    ) {
+      return;
+    }
+
+    const expense = await expensesService.create({
+      userId,
+      title,
+      spentAt,
+      category,
+      amount,
+      note,
+    });
+
+    res.status(201).send(expense);
+  } catch (error) {
+    res.status(500).send(INTERNAL_SERVER_ERROR);
```

Comment:
- Is it always `500` status and server error? For example, it can be some client error if the data is wrong
- Create a constant with all status codes